helios/management/commands/makeboothmessages.py

```diff
@@ -117,7 +117,8 @@ def copy_plural_forms(msgs, locale, domain, verbosity, stdout=sys.stdout):
     for domain in domains:
         django_po = os.path.join(django_dir, 'conf', 'locale', locale, 'LC_MESSAGES', '%s.po' % domain)
         if os.path.exists(django_po):
-            m = plural_forms_re.search(open(django_po, 'rU').read())
+            with open(django_po, 'rU') as f:
```

Comment:
`U` is deprecated and doesn't do anything in Python 3. Just use `open(django_po, 'r')`.